README.md

```diff
@@ -8,10 +8,14 @@ Substreams are consumable directly.
 
 ## Stream intermediate values and events
 
+- Replace the spkg with the version you would like to use, the current release can be found on [Github](https://github.com/streamingfast/substreams-uniswap-v3/releases).
+
+- Replace the StreamingFast api endpoint with the preferred network, a list of chains and endpoints can be found [here](https://substreams.streamingfast.io/reference-and-specs/chains-and-endpoints).
+
 ```bash
-$ substreams run https://github.com/streamingfast/substreams-uniswap-v3/releases/download/v0.1.0-beta/uniswap-v3-v0.1.0-beta.spkg \
+$ substreams run https://github.com/streamingfast/substreams-uniswap-v3/releases/download/v0.2.8/substreams.spkg \
   map_pools_created \
-  -e api-dev.streamingfast.io:443 \
+  -e api.streamingfast.io:443 \
```

Comment:
I just realized this should also be `mainnet.eth.streamingfast.io:443`, I'll change it after merging.

Thanks for the changes!